app/controllers/spree/api/v1/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,50 @@
+
+  class Spree::Api::V1::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+    include Spree::Core::ControllerHelpers::Common
+    include Spree::Core::ControllerHelpers::Order
+    include Spree::Core::ControllerHelpers::Auth
+    include Spree::Core::ControllerHelpers::Store
+    skip_before_action :verify_authenticity_token
+
+    def login
+      eligible_providers = SpreeSocial::OAUTH_PROVIDERS.map { |p| p[1] if p[2] == 'true' }.compact
```

Comment:
isn't there any helper method for this available yet? does not look like it really belongs to login method

Reply:
Would it make sense to use more sensible variable names instead of p?